### PR TITLE
[Feature-7465][Alert] Modifiy alert docs

### PR DESCRIPTION
**Pull request by SbloodyS**
related issue & PR:
https://github.com/apache/dolphinscheduler/issues/7460 
https://github.com/apache/dolphinscheduler/pull/7465
https://github.com/apache/dolphinscheduler/pull/7452

### CODE REVIEW

File: docs/en-us/dev/user_doc/guide/alert/enterprise-wechat.md [hunk 1]
```diff
@@ -1,29 +1,9 @@
 # Enterprise WeChat
 
-If you need to use Enterprise WeChat to alert, please modify the `alert.properties` file after the installation is complete, and then restart the alert service. The configuration example of enterprise WeChat is as follows
+If you need to use Enterprise WeChat to alert, please create alarm Instance in warning instance manage, and then chose the wechat plugin. The configuration example of enterprise WeChat is as follows
```
Comment (zhongjiajie) [inline docs/en-us/dev/user_doc/guide/alert/enterprise-wechat.md:3]:
```suggestion
If you need to use Enterprise WeChat to alert, please create an alarm Instance in warning instance manage, and then choose the wechat plugin. The configuration example of enterprise WeChat is as follows
```

File: docs/en-us/dev/user_doc/guide/alert/enterprise-wechat.md [hunk 2]
```diff
@@ -1,29 +1,9 @@
...
 
-```shell
-# Enable enterprise WeChat alarm or not: ``true`` or ``false``
-enterprise.wechat.enable="true"
+<img src="enterprise-wechat-plugin.png">
```
Comment (zhongjiajie) [inline docs/en-us/dev/user_doc/guide/alert/enterprise-wechat.md:5]: I thinks we should use markdown syntax here, `[enterprise-wechat-plugin](enterprise-wechat-plugin.png)`. WDYT
```suggestion
<img src="enterprise-wechat-plugin.png">
```

File: docs/en-us/dev/user_doc/guide/alert/enterprise-wechat.md [hunk 3]
```diff
@@ -1,29 +1,9 @@
...
-# Get the address of access_token, modify if you need
-enterprise.wechat.token.url=https://qyapi.weixin.qq.com/cgi-bin/gettoken?corpid={corpId}&corpsecret={secret}
-
-# Send application message address, modify if you need
-enterprise.wechat.push.url=https://qyapi.weixin.qq.com/cgi-bin/message/send?access_token={token}
-
-# Message format
-enterprise.wechat.user.send.msg={\"touser\":\"{toUser}\",\"agentid\":\"{agentId}\",\"msgtype\":\"markdown\",\"markdown\":{\"content\":\"{msg}\"}}
-```
+Where send Type corresponds to app and appchat respectively:
```
Comment (zhongjiajie) [inline docs/en-us/dev/user_doc/guide/alert/enterprise-wechat.md:7]:
```suggestion
Where send type corresponds to app and appchat respectively:
```

File: docs/en-us/dev/user_doc/guide/alert/enterprise-wechat.md [hunk 4]
```diff
@@ -1,29 +1,9 @@
...
+APP: https://work.weixin.qq.com/api/doc/90000/90135/90236
```
Comment (zhongjiajie) [inline docs/en-us/dev/user_doc/guide/alert/enterprise-wechat.md:8]: Is there a documnet in English? I think English is better but I could not find it?

**Review by zhongjiajie — CHANGES_REQUESTED** [review]

File: docs/en-us/dev/user_doc/guide/alert/enterprise-wechat.md (hunk 4 above)
Comment (SbloodyS) [reply docs/en-us/dev/user_doc/guide/alert/enterprise-wechat.md:8]: https://developers.weixin.qq.com/community/develop/doc/000ea2cdb84d9009c9ea7c7b651400
There is no English version of enterprise wechat API documents.

**Comment by SbloodyS** [comment]
@zhongjiajie Please review again.thanks.

File: docs/en-us/dev/user_doc/guide/alert/enterprise-wechat.md (hunk 4 above)
Comment (zhongjiajie) [reply docs/en-us/dev/user_doc/guide/alert/enterprise-wechat.md:8]: Ok, thanks for the clarification 👍 . But I still want to 👎  for wechat enterprise API doc

**Comment by zhongjiajie** [comment]
code looks good to me. I would merge this patch after CI pass. One more thing, I suggest to operate the unrelated doc next time to make our changelog more sense. In this PR, I prefer to submit two PR, one change wechat API doc, another change build-in parameter

**Comment by SbloodyS** [comment]
> code looks good to me. I would merge this patch after CI pass. One more thing, I suggest to operate the unrelated doc next time to make our changelog more sense. In this PR, I prefer to submit two PR, one change wechat API doc, another change build-in parameter

thanks.

**Comment by zhongjiajie** [comment]
I odd that CI would be cancel for no reason.

**Comment by SbloodyS** [comment]
> I was odd that CI would be canceled for no reason.

It seems stuck in Run for file in $(find . -name "*.md"); do. Maybe the CI server is busy and the command cannot be executed？

**Review by zhongjiajie — APPROVED** [review]
LGTM

**Comment by zhongjiajie** [comment]
It found a dead link which we just add to repo should could not be found, I think our dead link test should be improve to avoid this happend. https://github.com/apache/dolphinscheduler-website/runs/4639544230?check_suite_focus=true
image

**Pull request merged**